utils/build/docker/python/parametric/apm_test_client/server.py

```diff
@@ -136,6 +136,7 @@ def trace_config() -> TraceConfigReturn:
             "dd_trace_sample_ignore_parent": None,
             "dd_env": config.env,
             "dd_version": config.version,
+            "dd_trace_rate_limit": str(config._trace_rate_limit),
```

Comment:
Can we make this change to all the parametric apps. That way we can easily see which libraries pass this test and which don't. It should be a simple change in most libraries.

Reply:
Let me do it in a separate PR. I want to keep this PR scoped to the system test

Reply:
We should confirm whether this system test is written in a way where it is only compatible with python. It would be nice to know now if all/most libraries store the traceagent url in this format `http://random-host:9999/`. 

This isn't a blocker for this PR but moving forward, we should try to confirm the validity of the shared tests by running the test against 1 or more libraries